pkg/projector/slide/current_structure_level_list.go

```diff
@@ -75,6 +78,55 @@ func CurrentStructureLevelListSlideHandler(ctx context.Context, req *projectionR
 		})
 	}
 
+	var interventionSpeakers []dsmodels.Speaker
+	for _, speaker := range los.SpeakerList {
+		if speaker.SpeechState == "intervention" && speaker.EndTime == 0 {
+			interventionSpeakers = append(interventionSpeakers, speaker)
+		}
+	}
+
+	if len(interventionSpeakers) > 0 {
+		defaultInterventionTime, err := req.Fetch.Meeting_ListOfSpeakersInterventionTime(los.MeetingID).Value(ctx)
+		name := "\nIntervention"
+		if err != nil {
+			return nil, fmt.Errorf("couldn not load intervention time %w", err)
+		}
+		interventionEntry := structureLevelEntry{
+			Name:          name,
+			CountdownTime: float64(defaultInterventionTime),
+		}
+
+		var currentInterventionSpeaker *dsmodels.Speaker
+		for _, s := range interventionSpeakers {
+			if s.BeginTime != 0 && s.EndTime == 0 {
+				currentInterventionSpeaker = &s
+			}
+		}
+
+		if currentInterventionSpeaker != nil {
+			running := currentInterventionSpeaker.PauseTime == 0
+			interventionEntry.Running = running
+
+			if currentInterventionSpeaker.PauseTime == 0 {
+				interventionEntry.CountdownTime = float64(currentInterventionSpeaker.BeginTime) + float64(defaultInterventionTime) + float64(currentInterventionSpeaker.TotalPause)
+			} else {
+				now := int(time.Now().Unix())
+				elapsed := now - currentInterventionSpeaker.BeginTime - currentInterventionSpeaker.TotalPause
+				remaining := float64(defaultInterventionTime) - float64(elapsed)
+				interventionEntry.CountdownTime = remaining
+			}
```

Comment:
I think it might be a good idea to create a helper in `viewmodels` to calculate countdown times.